litmus-portal/graphql-server/graph/resolver.go

```diff
@@ -9,19 +9,21 @@ import (
 	"github.com/litmuschaos/litmus/litmus-portal/graphql-server/graph/generated"
 	"github.com/litmuschaos/litmus/litmus-portal/graphql-server/pkg/authorization"
 	"github.com/litmuschaos/litmus/litmus-portal/graphql-server/pkg/chaoshub"
+	"github.com/litmuschaos/litmus/litmus-portal/graphql-server/pkg/database/mongodb"
 )
 
 // This file will not be regenerated automatically.
 //
 // It serves as dependency injection for your app, add any dependencies you require here.
 
 type Resolver struct {
-	ChaosHubService chaoshub.Service
+	chaosHubService chaoshub.Service
 }
 
 func NewConfig() generated.Config {
+	dbOperator := &mongodb.MongoOperations{}
```

Comment:
This can be passed from the main() itself, we can share the same db instance and use it across all the services and go routines. We are already creating a db instance in the main for the recurring hub sync let's reuse that. In the future when all the services are refactored we will just have 1 single db instance passed around to all of them too.

Reply:
So, Instead of using a global variable like the below, Parameterize the db instance to `NewConfig(dbInstance)` and inject them in the main function. Is it right? @gdsoumya 


https://github.com/litmuschaos/litmus/blob/601da40aae7c3bae19fd5dc8fd4b133021e62f3d/litmus-portal/graphql-server/pkg/database/mongodb/init.go#L57

Reply:
Yup, we should move away from global variables and multiple separate initializations as much as we can.

Reply:
Okay I will figure it out 🚀